test/dummy/app/controllers/cache_controller.rb

```diff
@@ -1,5 +1,5 @@
 class CacheController < ApplicationController
-  before_filter :instrument_caching
+  Librato::Rails::VersionSpecifier.supported(max: '4.1') { before_filter :instrument_caching }
```

Comment:
Yeah, I was hoping that using a version specifier would convey that it is no longer supported, but your suggestion definitely makes sense so we know to remove it when/if we drop 4.1 support in the future. Will add a comment.